src/plugins/intel_cpu/src/nodes/eltwise.cpp

```diff
@@ -194,17 +219,19 @@ struct jit_uni_eltwise_generic : public jit_uni_eltwise_kernel, public jit_gener
             std::set<std::vector<element::Type>> prcs = get_supported_precisions(eltwise_data_[i].algo);
             std::set<std::vector<element::Type>> prcs_intersect = {};
 
-            // to support previous functionality
-            if (!std::all_of(
+            // to support previous functionality with one precision
```

Comment:
Please remove this comment or replace it on something like this: "Looking for precisions intersection between eltwise nodes."

Reply:
updated, but text is more sutable